cmd/scollector/conf/conf.go

```diff
@@ -50,6 +50,10 @@ type Conf struct {
 	// UseNtlm specifies if HTTP requests should authenticate with NTLM.
 	UseNtlm bool
 
+	// XAccessToken is an optional string that sets the X-Access-Token HTTP header
+	// which is used to authenticate against Bosun
+	XAccessToken string
```

Comment:
Lets call this `AuthToken` perhaps? Looks like maybe we've been a tad inconsistent.